crates/matrix-sdk-ui/src/timeline/polls.rs

```diff
@@ -37,17 +38,23 @@ pub(super) struct ResponseData {
 
 impl PollState {
     pub(super) fn new(content: NewUnstablePollStartEventContent) -> Self {
-        Self { start_event_content: content, response_data: vec![], end_event_timestamp: None }
+        Self {
+            start_event_content: content,
+            response_data: vec![],
+            end_event_timestamp: None,
+            has_been_edited: false,
```

Comment:
How important it is that this is correct? As far as I can tell, it's only set after an edit in the current session, but it would never inform you about edits done before starting the client, or done by other people.

Reply:
We use it to show "edited" in the UI, and we would like to see it at some point (ideally "live").

I made this try:
- Log in with the same user both on EX and Element Web
- Create a poll from Element Web
- View the poll from EX
- Edit the poll from Element Web
- EX sees the update correctly (`has_been_edited = true`)

So in which case you don't expect this to work?

Reply:
Ah perfect, thanks for trying it out, I didn't get this function was called upon receiving a remote echo :+1: